### PR TITLE
Icinga 2.12.10

### PR DESCRIPTION
**Pull request by Al2Klimov**
* Update CHANGELOG.md
* Bump ICINGA2_VERSION

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ documentation before upgrading to a new release.
 
 Released closed milestones can be found on [GitHub](https://github.com/Icinga/icinga2/milestones?state=closed).
 
+## 2.12.10 (2023-02-16)
+
+This release updates the bundled Boost and OpenSSL on Windows (#9686)
```
Comment (julianbrost) [inline CHANGELOG.md:12]: Should mentions the versions. One important reason for this release are security fixes in OpenSSL, so I'd say the new version is relevant here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ documentation before upgrading to a new release.
...
+and repairs broken SELinux policies (#9689).
```
Comment (julianbrost) [inline CHANGELOG.md:13]: This should be more detailed and state both things fixed, i.e. that on some systems, the policy couldn't be installed, on others, syslog inside checks run with sudo didn't work (if I understood those changes correctly).

**Review by julianbrost — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Al2Klimov) [reply CHANGELOG.md]: @htriem Please fix this for me and push on top.

Comment (Al2Klimov) [reply CHANGELOG.md]: As well as in #9692

Comment (htriem) [reply CHANGELOG.md]: Reworded in both PRs.

**Review by julianbrost — DISMISSED** [review]

**Comment by Al2Klimov** [comment]
Our GHA doesn’t cover changelog, so...

**Pull request merged**